dags.yaml

```diff
@@ -402,3 +402,23 @@ bqetl_release_criteria:
       - dthorn@mozilla.com
     retries: 2
     retry_delay: 30m
+
+bqetl_pocket:
+  default_args:
+    depends_on_past: false
+    email:
+      - jklukas@mozilla.com
+      - telemetry-alerts@mozilla.com
+    email_on_failure: true
+    email_on_retry: true
+    end_date: null
```

Comment:
I suppose this is something automatically added by the CLI. Might be nice to omit though in a future version.